github/table_github_organization_dependabot_alert.go

```diff
@@ -0,0 +1,299 @@
+package github
+
+import (
+	"context"
+
+	"github.com/google/go-github/v48/github"
+	"github.com/turbot/steampipe-plugin-sdk/v4/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func gitHubDependabotAlertColumns() []*plugin.Column {
+	return []*plugin.Column{
+		{
+			Name:        "dependabot_number",
```

Comment:
Is there a conflict with PostgreSQL reserved column names if we wanted to go with `number`?

Reply:
I don't think so. I looked at other tables and I found that the `github_issue` table has a `issue_number`  column. I tried to follow that, but it's easy to change that.

Reply:
In the [API](https://docs.github.com/en/rest/dependabot/alerts?apiVersion=2022-11-28#get-a-dependabot-alert) to get a specific dependabot alert, they use the param name `alert_number`, so maybe we should follow their example and rename this column to `alert_number`. I think this new column name is a bit more descriptive, as each row in this table represents an instance of an (dependabot) alert.

Reply:
Thanks @francois2metz for making the updates 👍.